lua/bufferline.lua

```diff
@@ -191,12 +196,12 @@ function bufferline.setup(user_config)
     {count = true, desc = 'Scroll the bufferline right'}
   )
 
-  -- Set the options and watchers for when they are edited
-  vim.g.bufferline = user_config or vim.empty_dict()
+  -- Setup barbar
+  events.enable()
+  vim.g.bufferline = user_config
 
-  highlight.setup()
-  JumpMode.set_letters(options.letters())
-  render.enable()
```

Comment:
`events.enable` ensures that setting `vim.g.bufferline` automatically calls these methods.